auk-notebook.ipynb

```diff
@@ -752,7 +855,7 @@
    "name": "python",
    "nbconvert_exporter": "python",
    "pygments_lexer": "ipython3",
-   "version": "3.7.1"
+   "version": "3.7.0"
```

Comment:
This is going to keep changing as different people have their hands on it. Not sure what to do here.